lib/livebook/runtime/node_pool.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Livebook.Runtime.NodePool do
+  use GenServer, async: false
```

Comment:
async: false is not known to GenServer. :) You also want to add @moduledoc false:

```suggestion
  @moduledoc false
  use GenServer
```